components/library/libxml2/patches/15.lib-run-path-fix.patch

```diff
@@ -0,0 +1,24 @@
+Make the library configuration universal for 32/64bits.
+
+Solaris specific. Not suitable for upstream.
+
+--- libxml2-2.10.2/xml2-config.in
++++ libxml2-2.10.2/xml2-config.in
+@@ -82,16 +82,7 @@
+        	;;
+ 
+     --libs)
+-        if [ "$2" = "--dynamic" ]; then
+-            shift
+-            libs="@XML_LIBS@"
+-        else
+-            libs="@XML_LIBS@ @XML_PRIVATE_LIBS@ @MODULE_PLATFORM_LIBS@ @LIBS@"
+-        fi
+-
+-        if [ "@XML_LIBDIR@" != "-L/usr/lib" -a "@XML_LIBDIR@" != "-L/usr/lib64" ]; then
+-            libs="@XML_LIBDIR@ $libs"
+-        fi
++        libs="@XML_LIBS@"
```

Comment:
This patch breaks other packages' configure scripts that may want to run "xml2-config --libs --dynamic". While it's fine to ignore --dynamic if it doesn't make sense anymore, it shouldn't be treated as an error if it was specified.